src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  background-color: $elements-bg-color;
+  color: $main-text-color;
+
+  transition: $transition-time;
```

Comment:
It would be great to specify the properties for transition for better performance. Now it applies to all properties